github/bump_versions/bump_versions.rb

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env ruby
+
+# This script bumps the version in all YaST packages
+#
+# You need admin access rights to temporarily disable the GitHub branch protection
+# and allow direct push to "master".
+#
+
+# the GitHub organization
+GH_ORG = "yast".freeze
+
+# install missing gems
+if !File.exist?("./.vendor")
+  puts "Installing the needed Rubygems to ./.vendor/bundle ..."
+  system "bundle install --path .vendor/bundle"
+end
+
+require "rubygems"
+require "bundler/setup"
+
+require "octokit"
+
+DRY_RUN = ENV["DRY_RUN"]
+
+# new package version
+NEW_PACKAGE_VERSION = "4.4.0".freeze
+# for checking whether the version has been already bumped
+NEW_PACKAGE_PREFIX = NEW_PACKAGE_VERSION[0..-2]
+
+# some packages use distro based version
+NEW_DISTRO_VERSION = "15.4.0".freeze
+NEW_DISTRO_PREFIX = NEW_DISTRO_VERSION[0..-2].freeze
+
+# subdirectory where to clone Git repositories
+GIT_CHECKOUT_DIR = "github".freeze
+
+# skip these repositories
+EXCLUDE_REPOS = [
+  # this is a Ruby gem, not an YaST module
+  "yast-rake",
+  # these are bound to the SUSE Manager version, not the SLE version
+  "skelcd-control-suse-manager-proxy",
+  "skelcd-control-suse-manager-server"
+].freeze
+
+def spec_files
+  Dir.glob("package/*.spec")
+end
+
+# read the package version from the spec file(s)
+def read_version
+  spec_files.map do |spec_file|
+    spec = File.read(spec_file)
+    spec.match(/^\s*Version:\s*(\S+)$/)[1]
+  end
+end
+
+# set the new version in the spec files(s)
+def set_version(version)
+  spec_files.each do |spec_file|
+    spec = File.read(spec_file)
+    spec.gsub!(/^\s*Version:.*$/, "Version:        #{version}")
```

Comment:
I used to use similar gsub but it sometimes breaks formatting, so I prefer `    spec.gsub!(/^(\s*)Version:(\s*).*$/, "\1Version:\2#{version}")` this way formatting survive intact